packages/client/mutations/UpdateReflectionGroupTitleMutation.ts

```diff
@@ -25,6 +26,16 @@ const mutation = graphql`
   }
 `
 
+export const updateReflectionGroupTitleMeetingUpdater = (payload: RecordProxy) => {
+  const reflectionGroupRecord = payload.getLinkedRecord('reflectionGroup')
+  if (!reflectionGroupRecord) return
+
+  const title = reflectionGroupRecord.getValue('title')
+  if (typeof title === 'string') {
+    reflectionGroupRecord.setValue(title, 'title')
```

Comment:
when relay receives a payload, it looks up the GUID for each record in the payload, matches it to a record in the sink (ie its store) and then updates the store record. it then discards the payload.

this fix just sets a payload value to itself, and since this operation doesn't touch the store, the payload is going to get disposed after calling this, so this function is a noop.

Reply:
Doh! Thanks, I'm now updating the store instead

Reply:
so relay already does this _exact_ thing whenever it receives a payload, just before this function gets called, so i'm hesitant to approve this PR since it's just executing the same code twice.

if you see an ellipsis, it can be traced back to this line: https://github.com/ParabolInc/parabol/blob/81043ad1a1265ca567778f95a5507cb5ae35734b/packages/client/components/ReflectionGroup/ReflectionGroupTitleEditor.tsx#L132

could we explore how dirtyRef is handled? is dirtyRef ever true for subscribers? also check out how a local client update is fired onChange, which would compete with the server update. if you wrap that in a setTimeout could you reproduce? i think the error is probably in that handling instead of in the relay internals